mcs/class/Mono.Profiler.Log/Mono.Profiler.Log/LogEvents.cs

```diff
@@ -276,7 +276,7 @@ public sealed class HeapRootsEvent : LogEvent {
 
 		public struct HeapRoot {
 
-			public long AddressPointer { get; internal set; }
+			public long SlotPointer { get; internal set; }
```

Comment:
Isn't this breaking change?

Reply:
Then just extend it, it's still better than maintaining two very similar support code

Reply:
The API of the library is explicitly not stable right now (see #6183) and the old name of this property was pretty bad.